### PR TITLE
NEXT-10889 - Change German translations for "valid", closes #1294

### LINKED ISSUE #1294: German "valide"
**Description**  
The german "Valide" is educational language.
We should move to "gültig".

Additionally, these two variants are mixed in use.

**Possible Solution**  
Adjust them all to one variant. Give me a info how you would like to have it, so I can create a PR.

**Additional context**

### PR DESCRIPTION
**Pull request by tinect**
### 1. Why is this change necessary?
The german "Valide" is educational language.
We should move to "gültig".

See #1294 

### 2. What does this change do, exactly?
- changed usage of "valid" to "gültig"

### 4. Please link to the relevant issues (if any).
https://issues.shopware.com/issues/NEXT-10889

### 5. Checklist

- [x] I have written tests and verified that they fail without my change
- [x] I have squashed any insignificant commits
- [x] I have created a [changelog file](https://github.com/shopware/platform/blob/master/adr/2020-08-03-Implement-New-Changelog.md) with all necessary information about my changes
- [x] I have written or adjusted the documentation according to my changes
- [x] This change has comments for package types, values, functions, and non-obvious lines of code
- [x] I have read the contribution requirements and fulfil them.

### CODE REVIEW

File: changelog/_unreleased/2020-09-16-change-german-translations-for-valid.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+title: Change German translations for "valid"
+issue: NEXT-10889
```
Comment (mitelg) [inline changelog/_unreleased/2020-09-16-change-german-translations-for-valid.md]: don't forget your name, github handle etc :wink:

Comment (tinect) [reply changelog/_unreleased/2020-09-16-change-german-translations-for-valid.md]: DAMN! :-)

**Comment by tinect** [comment]
@GitEvil Any news? :-)

**Comment by tinect** [comment]
@GitEvil ping

**Comment by philipreinken** [comment]
Thank you for your contribution @tinect :grin:

**Pull request closed without merging**